core/server/proxy/src/main/java/alluxio/proxy/s3/S3ObjectTask.java

```diff
@@ -560,11 +561,6 @@ public Response createObject(String objectPath, FileSystem userFs,
         } else {
           toRead = Long.parseLong(contentLength);
         }
-        try {
```

Comment:
interesting.. did we always support overwrite?

Reply:
yes, it's always overwrite in S3 semantics